game/project.godot

```diff
@@ -18,15 +18,23 @@ config/icon="res://icon.svg"
 [display]
 
 window/size/viewport_width=240
-window/size/viewport_height=240
+window/size/viewport_height=180
 window/size/window_width_override=1280
-window/size/window_height_override=1280
+window/size/window_height_override=1080
```

Comment:
Any particular reason why we changed this?

Reply:
This just scales the screen when you run the game

Reply:
I updated the window viewport size to 240x180 and then scaled it up to a larger window size. I didn't love the square window that we had...no other reason